<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reagents::CReagentWidget</class>
 <widget class="QWidget" name="Reagents::CReagentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="CReagentSubMenuWidget" name="reagentsubmenuPanel" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="reagentsStack">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="Reagents::CReagentStatusWidget" name="pageReagentStatus">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="Reagents::CReagentGroupWidget" name="pageReagentGroup"/>
     <widget class="Reagents::CReagentsWidget" name="pageReagents"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Reagents::CReagentStatusWidget</class>
   <extends>QWidget</extends>
   <header>Reagents/Include/ReagentStatusWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Reagents::CReagentGroupWidget</class>
   <extends>QWidget</extends>
   <header>Reagents/Include/ReagentGroupWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Reagents::CReagentsWidget</class>
   <extends>QWidget</extends>
   <header>Reagents/Include/ReagentsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CReagentSubMenuWidget</class>
   <extends>QWidget</extends>
   <header>Reagents/Include/ReagentSubMenuWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
